src/main/java/net/neoforged/neoforge/internal/NeoForgeStatesProvider.java

```diff
@@ -21,10 +22,11 @@ public class NeoForgeStatesProvider implements IModStateProvider {
     final ModLoadingState UNFREEZE = ModLoadingState.withInline("UNFREEZE_DATA", "INJECT_CAPABILITIES", ModLoadingPhase.GATHER, ml -> GameData.unfreezeData());
     final ModLoadingState LOAD_REGISTRIES = ModLoadingState.withInline("LOAD_REGISTRIES", "UNFREEZE_DATA", ModLoadingPhase.GATHER, ml -> GameData.postRegisterEvents());
     final ModLoadingState FREEZE = ModLoadingState.withInline("FREEZE_DATA", "LOAD_REGISTRIES", ModLoadingPhase.GATHER, ml -> GameData.freezeData());
+    final ModLoadingState REGISTER_TICKET_CONTROLLERS = ModLoadingState.withInline("REGISTER_TICKET_CONTROLLERS", "COMPLETE", ModLoadingPhase.COMPLETE, ml -> ForcedChunkManager.init());
```

Comment:
We should have a generic place to fire all forge common registration events (e.g. capabilities after the rework as well) instead of adding more state transitions.